### PR TITLE
ADD README.md for TPU_v4 user guide

### PR DESCRIPTION
**Pull request by tonyjohnchen**

### CODE REVIEW

File: Training/TPU-v5p/Llama2-7B-Maxtext/TPU_v4_user_guide.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
 # Instructions for training Llama2-7B-Maxtext on TPU v4
```
Comment (RissyRan) [inline Training/TPU-v5p/Llama2-7B-Maxtext/TPU_v4_user_guide.md:1]: Should this be under `Training/TPU-v4/Llama2-7B-Maxtext/README.md`?

File: Training/TPU-v5p/Llama2-7B-Maxtext/TPU_v4_user_guide.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+    <td>1:1</td>
+  <tr>
+    <td>V5p/Viperfish</td>
+    <td>v5p-8</td>
+    <td>v2-alpha-tpuv5</td>
+    <td>2:1</td>
+  </tr>
+</table>
+
+* Note: On cloud, we use <TPU_version>-<Core_number> for TPU type. For example, if we create v4-8 TPU, 8 is the number of cores, and it uses 4 chips, since Cores:Chips is 2:1 for v4.
```
Comment (RissyRan) [inline Training/TPU-v5p/Llama2-7B-Maxtext/TPU_v4_user_guide.md:31]: I feel those content is kind of mixed. This is for v4 user guide, and it also basic info for v5e, v5p, and v4 (general info, not model specific).

**Review by RissyRan — COMMENTED** [review]
Thanks Tony!

File: Training/TPU-v5p/Llama2-7B-Maxtext/TPU_v4_user_guide.md (hunk 2 above)
Comment (tonyjohnchen) [reply Training/TPU-v5p/Llama2-7B-Maxtext/TPU_v4_user_guide.md:31]: These are the topics I want to go over for this coming Wednesday training. I will update the content once the full user guide is complete.

**Review by stingram — CHANGES_REQUESTED** [review]
Please address comments by @RissyRan

File: Training/TPU-v5p/Llama2-7B-Maxtext/TPU_v4_user_guide.md (hunk 1 above)
Comment (tonyjohnchen) [reply Training/TPU-v5p/Llama2-7B-Maxtext/TPU_v4_user_guide.md:1]: Done.

**Review by RissyRan — DISMISSED** [review]
Thanks! LGTM!

File: Training/TPU-v5p/Llama2-7B-Maxtext/README.md [hunk 3]
```diff
@@ -1 +1,3 @@
 # Instructions for training Llama2-7B-Maxtext on TPU v5p
+
+Please check the v4 user guide first: https://github.com/gclouduniverse/reproducibility/blob/main/Training/TPU-v4/Llama2-7B-Maxtext/TPU_v4_user_guide.md
```
Comment (RissyRan) [inline Training/TPU-v5p/Llama2-7B-Maxtext/README.md:3]: Nit: 

Please check the v4 user guide first: [here](https://github.com/gclouduniverse/reproducibility/blob/main/Training/TPU-v4/Llama2-7B-Maxtext/TPU_v4_user_guide.md)

Comment (tonyjohnchen) [reply Training/TPU-v5p/Llama2-7B-Maxtext/README.md]: Done.

**Comment by tonyjohnchen** [comment]
> Please address comments by @RissyRan

Done.

**Comment by tonyjohnchen** [comment]
> Please address comments by @RissyRan

Done.

**Review by stingram — APPROVED** [review]

**Pull request merged**